src/angular-gridster.js

```diff
@@ -38,8 +41,8 @@
 		}
 	})
 
-	.controller('GridsterCtrl', ['gridsterConfig', '$timeout',
-		function(gridsterConfig, $timeout) {
+	.controller('GridsterCtrl', ['gridsterConfig', '$timeout', '$rootScope',
```

Comment:
why wouldn't you `$emit` an event from the current `$scope`?
if the page has multiple gridsters, each with an unplaceable listener would fire.